.github/workflows/schedule.py

```diff
@@ -167,21 +174,38 @@ def create_issue(self, title_date: date) -> None:
         if issues:
             log.info('At least one matching issue already exists: %r', issues)
         else:
-            assignees = self.properties.get('assignees')
-            if assignees is not None:
-                flags.append(f'--assignee={assignees}')
             command = [
                 'gh', 'issue', 'create',
                 *flags,
                 f'--title={title}',
-                f'--label={self.properties["labels"]}',
                 f'--body={self.body}'
             ]
+            edit_command, flags = ['gh', 'issue', 'edit', *flags], []
```

Comment:
This resets flags, thereby discarding the `--repo` parameter but I think the edit command requires it. How do you test your changes?

Reply:
My question wasn't rhetorical. Given that I was under the (wrong) impression that your code was incorrect, it is understandable that I ask for your test methodology.